base/rational.jl

```diff
@@ -375,8 +375,8 @@ function ^(x::Rational, n::Integer)
 end
 
 ^(x::Number, y::Rational) = x^(y.num/y.den)
-^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T, y.num / y.den))
-^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T, y.num / y.den))
+^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
+^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
```

Comment:
`convert(T,y)` is safe, because there is a [generic conversion from Rational to AbstractFloat](https://github.com/JuliaLang/julia/blob/adecd7323455f09462588667e225b6467099e6a1/base/rational.jl#L71-L74).